README.md

```diff
@@ -175,12 +175,12 @@ This will build all library components as well as 'mlpack_test'.
 
     $ make
 
-You can specify individual components which you want to build, if you do not
+You can specify individual components which you want to build if you do not
```

Comment:
This is not grammatically incorrect, and neither are the changes made below. Please revert.